eu.skribisto.skribisto.yml

```
@@ -0,0 +1,37 @@
+app-id: eu.skribisto.skribisto
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: skribisto
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+separate-locales: false
+modules:
+        
+  - name: quazip
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=MinSizeRel
+    sources:
+      - type: archive
+        url: https://github.com/stachenov/quazip/archive/v1.1.tar.gz
+        sha256: 54edce9c11371762bd4f0003c2937b5d8806a2752dd9c0fd9085e90792612ad0
+      - type: shell
+        commands:
+          - sed -i 's|${CMAKE_ROOT}/Modules|share/cmake|' CMakeLists.txt
+
+  - name: skribisto
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=MinSizeRel
```

```suggestion
      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Debug symbols are split to separate ref and are not installed by default (but can be for debugging).

---

Thank you !